src/aks-preview/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open as open1
 from setuptools import setup, find_packages
 
-VERSION = "0.4.57"
+VERSION = "0.4.58"
```

Comment:
Also need to add an item in HISTORY.md

Reply:
Can we keep the version bumps and release PRs separate from feature PRs?

It's easier when writing release notes so that we make sure we don't miss anything.

It also makes it easier to revert the feature work in case something is wrong.

Reply:
Thanks for reviewing, I reverted the version change in `setup.py` and will send out a separate PR after this PR is merged.